src/app/beer_garden/api/stomp/server.py

```diff
@@ -0,0 +1,150 @@
+import sys
+import stomp
+import logging
+import time
+import beer_garden.config as config
+from brewtils.models import Event, Events, Request, Operation, System
+from brewtils.schema_parser import SchemaParser
+import beer_garden.events
+import beer_garden.router
+from beer_garden.api.stomp.processors import append_headers, process_send_message
+
+
+conn = None
+bg_active = False
+
+
+def send_message(message, headers={}):
+    global conn
+    stomp_config = config.get("entry.stomp")
+    message, response_headers = process_send_message(message)
+    response_headers = append_headers(
+        response_headers=response_headers, request_headers=headers
+    )
+    if conn.is_connected():
+        if "reply-to" in headers:
+            conn.send(
+                body=message, headers=response_headers, destination=headers["reply-to"]
+            )
+        else:
+            conn.send(
+                body=message,
+                headers=response_headers,
+                destination=stomp_config.event_destination,
+            )
+
+
+def send_error_msg(error, headers):
+    global conn
+    stomp_config = config.get("entry.stomp")
+    error_headers = None
+    error_headers = append_headers(error_headers, headers)
+
+    if conn.is_connected():
+        if "reply-to" in headers:
+            conn.send(
+                body=error.__str__(),
+                headers=error_headers,
+                destination=headers["reply-to"],
+            )
+        else:
+            conn.send(
+                body=error.__str__(),
+                headers=error_headers,
+                destination=stomp_config.event_destination,
+            )
+    pass
+
+
+class OperationListener(stomp.ConnectionListener):
+    def on_error(self, headers, message):
+        print("received an error:", headers)
```

Comment:
We do not use Print statements in our baseline. You should use the logger like you did in the Connection class lower in the file.